### DISCUSSION TITLE
Where does Sonixd fetch lyrics

### DISCUSSION THREAD

**Discussion by ElMarco99** [jeffvli/sonixd#433] [Q&A] [answered]
Hi everyone, and thanks for the incredible airsonic client!
I've found myself wondering why most of my songs don't show lyrics (even though are incorporated as a tag), and some of those show the lyrics but are wrong. So, i'm guessing Sonixd is fetching lyrics from the net. Where can i change the setting? I can't find it

**Comment by dweymouth** [comment, marked as answer]
I am pretty sure Sonixd doesn't have any of its own lyric fetching built in. Instead the lyrics come from the server via the `getLyrics` API for Subsonic (and the equivalent API for Jellyfin - I'm not familiar with Jellyfin's API). Depending on your server, they may fetch lyrics in different ways. Navidrome for example only fetches embedded lyrics at the moment.

**Comment by ElMarco99** [comment]
I see, too bad i have airsonic server side. Maybe i'll install Navidrome
